src/libraries/sonnet/src/core/sonnet-core.pri

```diff
@@ -35,6 +35,29 @@ HEADERS += $$PWD/client_p.h \
 # Sonnet Plugins
 include($$PWD/../plugins/hunspell/hunspell.pri)
 
+#linux / mac
+CONFIG(with_aspell) {
+unix: {
+    exists("/usr/include/aspell.h") {
+        DEFINES += ASPELL_ENABLED
+        message ("ASpell enabled")
+        include($$PWD/../plugins/aspell/aspell.pri)
+        LIBS += -laspell
+    }
+}
+}
+
+#windows
+CONFIG(with_aspell) {
+win32: {
+    exists ("C:\\Qt\\5.13\\mingw73_32\\include\\aspell.h") {
```

Comment:
That will break on AppVeyor as soon Qt 5.13 will not be used any more. And for the rest of the users it will probably never work. :grimacing:

Reply:
Yes, this was just for the sake of completeness. :grimacing: 
It isn't supposed to work lol :grimacing: 

Aspell on windows has poor support.

Reply:
Ok :)